src/components/Footer.js

```diff
@@ -1,111 +1,23 @@
 import React from 'react'
-import { Link } from 'gatsby'
+import logo from '../../static/img/vocal-arts-logo-full-color.svg'
 
-import logo from '../img/logo.svg'
-import facebook from '../img/social/facebook.svg'
-import instagram from '../img/social/instagram.svg'
-import twitter from '../img/social/twitter.svg'
-import vimeo from '../img/social/vimeo.svg'
+import './Footer.scss'
 
 const Footer = class extends React.Component {
+  getYear() {
+    return new Date().getFullYear();
+  }
+  
   render() {
     return (
-      <footer className="footer has-background-black has-text-white-ter">
-        <div className="content has-text-centered">
+      <footer className="footer">
+        <div className="footer__content">
           <img
             src={logo}
-            alt="Kaldi"
-            style={{ width: '14em', height: '10em' }}
+            alt="Clemson Vocal Arts logo"
+            style={{ width: '15rem', height: 'auto' }}
```

Comment:
Since we already have a `.scss` file associated with the footer lets put this styling in there.